examples/booking_app/lib/feature_hotel/search/blocs/hotel_list_bloc.dart

```diff
@@ -46,11 +53,33 @@ class HotelListBloc extends $HotelListBloc {
     PaginatedHotelsRepository repository,
     CoordinatorBlocType coordinatorBloc,
   ) {
+    _$filterByQueryEvent
```

Comment:
@DDavidPrime, considering is a `Subject` is there a real benefit to creating another subject such as  `_querySubject`

Reply:
One of the golden rules of the Rx World is the less subjects the better